docker/docker_client.go

```diff
@@ -51,7 +52,10 @@ const (
 	backoffMaxDelay      = 60 * time.Second
 )
 
-var systemPerHostCertDirPaths = [2]string{"/etc/containers/certs.d", "/etc/docker/certs.d"}
+var (
+	homeCertDir               = filepath.FromSlash(".config/containers/certs.d")
+	systemPerHostCertDirPaths = []string{"/etc/containers/certs.d", "/etc/docker/certs.d"}
```

Comment:
are these documented anywhere?

Reply:
Good point, updating `docs/containers-certs.d.5.md` would be nice.

Reply:
container-certs.d.conf has info.